src/tools/run-make-support/src/diff/tests.rs

```diff
@@ -0,0 +1,38 @@
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_diff() {
+        let expected = "foo\nbar\nbaz\n";
+        let actual = "foo\nbar\nbaz\n";
+        diff().expected_text(expected).actual_text(actual).run();
+    }
+
+    #[test]
+    #[should_panic]
+    fn test_do_run() {
```

Comment:
I believe this test case for `panic` is usable. If in the future the `diff` run fails but does not trigger a `panic`, then all the test cases written using this tool will become ineffective.

Reply:
That makes sense to me yes.